### PR TITLE
fix path of git sha1 in config.yml

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Must have
- [x] Tests

### CODE REVIEW

File: demisto_sdk/commands/validate/file_validator.py [hunk 1]
```diff
@@ -1129,7 +1129,7 @@ def get_content_release_identifier(self):
         except Exception:
             return
         else:
-            return file_content.get('jobs').get('build').get('environment').get('GIT_SHA1')
+            return file_content.get('references').get('environment').get('environment').get('GIT_SHA1')
```
Comment (Itay4) [inline demisto_sdk/commands/validate/file_validator.py:1132]:
```suggestion
            return file_content.get('references', {}).get('environment', {}).get('environment', {}).get('GIT_SHA1')
```

File: demisto_sdk/commands/validate/validate_manager.py [hunk 2]
```diff
@@ -954,7 +954,7 @@ def get_content_release_identifier(self):
         except Exception:
             return
         else:
-            return file_content.get('jobs').get('build').get('environment').get('GIT_SHA1')
+            return file_content.get('references').get('environment').get('environment').get('GIT_SHA1')
```
Comment (Itay4) [inline demisto_sdk/commands/validate/validate_manager.py:957]: we shouldnt have 2 funcs that do the exact same thing.. lets use the same one

File: demisto_sdk/commands/validate/file_validator.py (hunk 1 above)
Comment (jochman) [reply demisto_sdk/commands/validate/file_validator.py:1132]: Don't you think it'll be better if it actually fails if it fails in this step?

Comment (Itay4) [reply demisto_sdk/commands/validate/file_validator.py:1132]: on `NoneType has no get`? no..

Comment (jochman) [reply demisto_sdk/commands/validate/file_validator.py]: yeah, but it means that it got the circle.yml as it needs to then the key is wrong.

Comment (jochman) [reply demisto_sdk/commands/validate/file_validator.py]: more likely is that we will encounter it before it'll reach any customer

Comment (Itay4) [reply demisto_sdk/commands/validate/file_validator.py]: doesnt matter.. if something is wrong, need to have an indicative error message

**Pull request merged**